samples/cxx/combustor/combustor.cpp

```diff
@@ -68,15 +68,15 @@ void runexample()
     // a short Gaussian pulse only to ignite the mixture
     MassFlowController m1;
     m1.install(fuel_in, combustor);
-    m1.setMassFlowRate(fuel_mdot);
+    m1.setMassFlowCoeff(fuel_mdot);
```

Comment:
I hadn't noticed this when bc8b4be6542a625a641a29169d308973de3a2c63 was added, but I think it would make sense for `MassFlowController` to override `setMassFlowRate`, since that is explicitly what is meant for this flow device -- the flow rate isn't this coefficient times some other quantity.

Reply:
This wasn't as apparent from the cython interface, but I named the parameter `m_coeff` to handle both constant and time-varying cases. From C++, I agree that `setMassFlowRate` is more intuitive, and re-introduced it for the device-specific interface (i.e. `MassFlowController::setMassFlowRate` overloads the deprecated method).

I also tweaked the cython interface, and introduced `mass_flow_rate` as a property (to replace the `set_mass_flow_rate` method)